VSDiagnostics/VSDiagnostics/VSDiagnostics.Test/Tests/General/SingleEmptyConstructorTests.cs

```diff
@@ -257,5 +257,46 @@ public MyExceptionClass() : this()
 
             VerifyDiagnostic(original);
         }
+
+        [TestMethod]
+        public void SingleEmptyConstructor_ConstructorHasXmlDocComment_ThisKeyword_DoesNotInvokeWarning()
```

Comment:
can you remove "_ThisKeyword" from this and the below one? It's not relevant here